firedrake/preconditioners/asm.py

```diff
@@ -234,7 +235,11 @@ def get_patches(self, V):
             indices = []
             for (i, W) in enumerate(V):
                 section = W.dm.getDefaultSection()
-                for p in pt_array:
+                if i == exclude_subspace:
```

Comment:
Will we ever want to exclude more than one subspace?

Reply:
We certainly might.. Fixed now